client/src/JobPaginatorZoom.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import PaginatorPreview from "./PaginatorPreview";
+import PaginatorZoomy from "./PaginatorZoomy";
+
+const JobPaginatorZoom = ({ zoom, getImageUrl }) => {
+    return (
+        <div className="six col">
```

Comment:
I also wonder whether it's better to factor this outermost `<div>` back into the Job Paginator itself... since this is part of the paginator's layout, it seems like it might be cleaner to keep the layout columns together in the same component so it's easier to adjust them, rather than embedding them in this component (which reduces its ability to be used in a different context).

Reply:
I think I should have kept it in the job paginator after reviewing the code.

Reply:
actually, I'll move the column

Reply:
Made change.